README.md

```diff
@@ -36,27 +36,27 @@ To develop a specific site, `cd` into the relevant sub-directory and follow the
 1. In the root of this repo, or in your own repo, run:
 
 ```bash
-hugo new site org-<your-org-name>
-cd org-<your-org-name>
+hugo new site org-{your-org-name}
+cd org-{your-org-name}
 ```
 
 2. Initialise your new site as a hugo module, as only modules can import modules:
 
 ```zsh
-hugo mod init github.com/CodeYourFuture/curriculum/org-<your-org-name>
+hugo mod init github.com/{your-org}/{your-repo}/{your-module}
 ```
 
 Then add the common theme and content modules as hugo modules to hugo.toml:
 
 ```toml
 [module]
   [[module.imports]]
-    path = "github.com/CodeYourFuture/curriculum/common-theme"
+    path = 'github.com/CodeYourFuture/curriculum/common-theme'
```

Comment:
Just curiosity: Is there a reason to prefer single quotes in this kind of context? I am a strong "double quotes by default" person (because it's more often that I want a `'` inside a quoted string than a `"`)...

Reply:
Oh no sorry that's my inattention. I changed them to test whether I needed to htmlUnescape, but I didn't, I just had vendored my modules in common-docs and wasn't getting my updates.